prov/rxm/src/rxm_ep.c

```diff
@@ -1138,6 +1135,13 @@ static int rxm_ep_bind(struct fid *ep_fid, struct fid *bfid, uint64_t flags)
 			if (ret)
 				return ret;
 		}
+		ret = ofi_ep_bind_cq(&rxm_ep->util_ep, cq, flags);
+		if (ret) {
+			if (cq->wait && ofi_wait_fd_del(cq->wait, rxm_ep->msg_cq_fd))
+				FI_INFO(&rxm_prov, FI_LOG_EP_CTRL,
+					"Unable to delete wait fd from FD list");
+			return ret;
+		}
```

Comment:
Is this change related to #3452?  Or is this part of the resource leak cleanup?

Reply:
It's part of the resource leak cleanup.
Before I've upstremed #3452, I had changes to the AV binding to the EP. I've dropped them